src/components/BoardControls/BoardControls.tsx

```diff
@@ -1,49 +1,34 @@
 import * as React from "react";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
-import { faPencilAlt, faCog } from "@fortawesome/free-solid-svg-icons";
+import { faPencilAlt } from "@fortawesome/free-solid-svg-icons";
 
+import "./board-controls.css";
 interface BoardControlsProps {
   addColumn: () => void;
   title: string;
   description: string;
   socket: SocketIOClient.Socket;
   boardId: string;
+  remainingVotes: number | undefined;
 };
 
 interface BoardControlsState {
-  title: string;
-  description: string;
   isEditingTitle: boolean;
 }
 
 export class BoardControls extends React.Component<BoardControlsProps, BoardControlsState> {
   private titleInput: React.RefObject<HTMLInputElement>;
+  private descriptionInput: React.RefObject<HTMLInputElement>;
```

Comment:
I know I'm the one that added it but: thoughts on removing description entirely from the boards? It seems sort of superfluous and adds complexity where it might not be needed. I originally included it because FunRetro has it but it might be just easier to kill it off.